packages/md-enhance/src/shared/options.ts

```diff
@@ -232,13 +233,22 @@ export interface MarkdownEnhanceOptions {
    */
   presentation?: PresentationOptions | boolean;
 
-  /**
+  /*
```

Comment:
Do not change this, comment starting with `/**` has special meanings

Reply:
I missed 1 `*` when merging...